cmd/start/start.go

```diff
@@ -0,0 +1,81 @@
+package start
+
+import (
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+
+	"github.com/MonikaCat/ibcjuno/utils"
+	workerctx "github.com/MonikaCat/ibcjuno/worker"
+	"github.com/rs/zerolog/log"
+
+	types "github.com/MonikaCat/ibcjuno/cmd/start/config"
+	"github.com/spf13/cobra"
+)
+
+var (
+	waitGroup sync.WaitGroup
+)
+
+// NewStartCmd returns the command that is run when starting IBCJuno
+func NewStartCmd(cmdCfg *types.StartConfig) *cobra.Command {
+	return &cobra.Command{
+		Use:     "start",
+		Short:   "Start IBCJuno price aggregator",
+		PreRunE: types.ReadConfigPreRunE(cmdCfg),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			context, err := types.GetStartContext(utils.Cfg, cmdCfg)
+			if err != nil {
+				return err
+			}
+
+			return StartIBCJuno(context)
+		},
+	}
+}
+
+// StartIBCJuno represents the function that is called when
+// "start" command is executed
+func StartIBCJuno(ctx *workerctx.WorkerContext) error {
+
+	// create worker responsible for fetching latest prices
+	worker := workerctx.NewWorker((ctx))
```

Comment:
Double parenthesis here shouldn't be needed 
```suggestion
	worker := workerctx.NewWorker(ctx)
```